app/templates/components/player-hud-components/player-infos.hbs

```diff
@@ -2,14 +2,15 @@
   <div class="img_container">
     <img src={{me.user.img_url}} />
   </div>
-  <div class="flex">
-    <div class="exp_level">Exp. level {{me.user.exp_level}}</div>
-  </div>
-  <div class="exp-meter">
-    <div class="exp-meter-progress"></div>
-    <div class="exp-meter-text">
-      {{me.user.experience}} / {{me.user.needed_exp_for_next_level}}
+
+  <div class="level-info-container__right">
+    <div class="exp_level">Exp. level
+      {{#lr-progress}}
+        template block text
+      {{/lr-progress}}
+      {{!-- {{me.user.exp_level}} --}}
     </div>
+    {{!-- {{lr-progress reached=me.user.experience total=me.user.needed_exp_for_next_level success=true}} --}}
```

Comment:
I guess you forgot to remove the upper part and to uncomment the part below ? If you included the upper part for trying to get the tests running, don't worry about that. I will fix the tests, as soon as you'll have comitted the other changes.